src/excmds.ts

```diff
@@ -1456,7 +1457,9 @@ export async function tablast() {
 
 /** Like [[open]], but in a new tab. If no address is given, it will open the newtab page, which can be set with `set newtab [url]`
 
-    Use the `-b` flag as the first argument to open the tab in the background.
+    Use the `-c` flag followed by a ccontainer id to open a tab in said container.
```

Comment:
1. Container is spelt with a single c :)
2. Please mention that the match is fuzzy.